eth/db/chain_gaps.py

```diff
@@ -48,7 +110,7 @@ def calculate_gaps(newly_persisted: BlockNumber, base_gaps: ChainGaps) -> GapInf
             raise GapTrackingCorrupted(
                 "Corrupted chain gap tracking",
                 f"No. {newly_persisted} appears to be missing in multiple gaps",
-                f"1st gap goes from {matching_gaps[0][1][0]} to {matching_gaps[0][1][1]}"
+                f"1st gap goes from {matching_gaps[0][1][0]} to {matching_gaps[0][1][1]}",
```

Comment:
Looking at these tripple indexes makes my eyes melt a little.  Is there a `NamedTuple` type we could use to reduce some of the cognitive overhead here?  Something like ....

```python
class Gap(NamedTuple):
    lower_bound: BlockNumber
    upper_bound: BlockNumber
```